proposals/lip-0045.md

```diff
@@ -984,166 +886,364 @@ The Interoperability module contains the following commands:
 * [State recovery initialization command][lip-0054]
 * [Message recovery initialization command][lip-0054]
 
-### Protocol Logic for Other Modules
+### Cross-chain Commands
 
-#### send
+The Interoperability module contains the following cross-chain commands:
 
-For the function description, parameters, and return value see the corresponding internal function `sendInternal`.
+* [Sidechain terminated cross-chain command][lip-0049]
+* [Registration cross-chain command][lip-0049]
+* [Channel terminated cross-chain command][lip-0049]
+  
+### Events
 
-##### Execution
+#### ChainAccountUpdated
 
-```python
-send(feeAddress, moduleID, crossChainCommandID, receivingChainID, fee, status, params):
-    return sendInternal(
-        feeAddress,
-        moduleID,
-        crossChainCommandID,
-        receivingChainID,
-        fee,
-        status,
-        params
-    )
-```
+This event has `name = EVENT_NAME_CHAIN_ACCOUNT_UPDATED`. 
+This event is emitted when a chain account is updated.
 
-#### error
+##### Topics
 
-The `error` function is used to add an error code to a CCM and then add it to the outbox of a partner chain.
+- `chainID`: The ID of the chain.
 
-##### Parameters
+##### Data
 
-This function has the following input parameters in the order given below:
+The `data` property of this event uses the `mainchainRegParams` schema given in [LIP0043][lip-0043].
```

Comment:
Why does it use `mainchainRegParams` schema? Shouldn't it be `chainAccountSchema`? In the pseudo code above for sidechain registration it uses chainAccount as data.

Reply:
I switched the two events! fixed